docs/production/self-hosting/workflow-recovery.md

```diff
@@ -0,0 +1,38 @@
+---
+sidebar_position: 50
+title: Workflow Recovery
+---
+
+When the execution of a durable workflow is interrupted (for example, if its executor is restarted, interrupted, or crashes), another executor must recover the workflow and resume its execution.
+To prevent duplicate work, it is important to detect interruptions promptly and to recover each workflow only once.
+This guide describes how to manage workflow recovery in a production environment.
+
+## Managing Recovery
```

Comment:
imho the ordering of these sections is repeating the mistake of not putting Transact first.

- Single server recovery in Transact
- Multiple servers recovery in Transact -- and key concerns
- Recovery with Conductor -- and how it handles the key concerns above